autometa/taxonomy/ncbi.py

```diff
@@ -364,60 +347,68 @@ def parse_merged(self):
             logger.debug('merged loaded')
         return merged
 
-    def is_common_ancestor(self, parent_taxid, child_taxid):
-        """Determines whether the provided taxids have a non-root common ancestor
+    def is_common_ancestor(self, taxid_A, taxid_B):
+        """
+        Determines whether the provided taxids have a non-root common ancestor
 
         Parameters
         ----------
-        parent_taxid : int
-            Description of parameter `parent_taxid`.
-        child_taxid : int
-            Description of parameter `child_taxid`.
+        taxid_A : int
+            identifer for a taxon in the Taxonomy Database by NCBI
+        taxid_B : int
+            identifer for a taxon in the Taxonomy Database by NCBI
```

Comment:
What database(s)?

Reply:
I am not exactly clear about this comment.
Should I specify, from nodes.dmp, names.dmp or merged.dmp ?

Reply:
You are saying both `taxid_A` and `taxid_B` are "identifiers for a taxon in the Taxonomy Database by NCBI"

However, what _is_ the Taxonomy Database by NCBI? Is this referring to one file? Multiple files? There should be some information here to tell the developer where the identifier originates.